common/src/main/java/io/netty/util/internal/MpscLinkedQueueNode.java

```diff
@@ -18,6 +18,10 @@
 
 import java.util.concurrent.atomic.AtomicReferenceFieldUpdater;
 
+/**
+ * @deprecated will be removed as not used anymore inside netty.
```

Comment:
Because it is used by OneTimeTask and I know projects use OneTimeTask. The idea was to make it as smooth as possible to upgrade.

> Am 28.05.2016 um 11:33 schrieb Johno Crawford <notifications@github.com>:
> 
> In common/src/main/java/io/netty/util/internal/MpscLinkedQueueNode.java:
> 
> > @@ -18,6 +18,10 @@
> >  
> >  import java.util.concurrent.atomic.AtomicReferenceFieldUpdater;
> >  
> > +/**
> > + * @deprecated will be removed as not used anymore inside netty.
> Then why not just remove it? It is in the internal package after all.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>